gi_service/contract/rest/api_contract.yaml

```
@@ -0,0 +1,782 @@
+openapi: 3.1.0
+
+info:
+  title: OpenAPI General Information Service
+  description: This document gives the structure of APIs for the GI Service
+  version: 1.0.0
+
+servers:
+- url: http://localhost:8081
+
+tags:
+- name: Organization
+  description: APIs for organization
+- name: Data
+  description: APIs for data
+
+paths:
+  /all-presidents:
+    get:
+      tags:
+      - Organization
+      summary: Get all president list with terms
+      description: Returns the list of presidents and terms
+      operationId: getPresidents
+      responses:
+        "200":
+          description: Successfully returned
```

Should this be simply.

```suggestion
          description: OK
```

---

I am referring to every 200 response in each endpoint.